src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -251,6 +251,7 @@ class FlowRunProperties:
     # Experiment command node fields only
     COMMAND = "command"
     OUTPUTS = "outputs"
+    RESUME_FROM = "resume_from"
```

Comment:
For pfazure Run, are these properties stored in RunHistory property?

Reply:
Offline synced.